site/_includes/layouts/devsite.njk

```diff
@@ -132,6 +126,18 @@
     </div>
     {% include 'partials/cookie-banner.njk' %}
 
-    {% block scripts %}{% endblock %}
+    {% block scripts %}
+    
+    <script>
+        const button = document.querySelector('.dcc-hero .button');
```

Comment:
This script prevents from the anchor to get attached in the url bar.
Please use css rule scroll-behavior: smooth; instead and reove unnecessary js.